README.md

```diff
@@ -43,6 +43,7 @@
 $ git clone git@github.com:luanfonceca/speakerfight.git
 $ cd speakerfight
 $ make setup
+$ make db
```

Comment:
This change is not necessary for two reasons:
1. The `make setup` runs the `db`.
2. If this was necessary it should be in other PR